client/src/components/HistoryViewer/tests/HistoryViewerHeading-test.js

```diff
@@ -1,32 +1,48 @@
+/* eslint-disable import/no-extraneous-dependencies */
 /* global jest, describe, it, expect */
 
 import React from 'react';
-import ReactTestUtils from 'react-addons-test-utils';
-import HistoryViewerHeading from '../HistoryViewerHeading';
-
-class HeadingWrapper extends React.PureComponent {
-  render() {
-    return (
-      <table>
-        <thead>
-          <HistoryViewerHeading {...this.props} />
-        </thead>
-      </table>
-    );
-  }
-}
+import { Component as HistoryViewerHeading } from '../HistoryViewerHeading';
+import Enzyme, { shallow } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-15.4/build/index';
+
+Enzyme.configure({ adapter: new Adapter() });
 
 describe('HistoryViewerHeading', () => {
-  it('has four columns when hasActions is true', () => {
-    const component = ReactTestUtils.renderIntoDocument(
-      <HeadingWrapper hasActions />
-    );
+  // Mock select functions to replace the ones provided by mapDispatchToProps
+  const mockOnCompareModeSelect = jest.fn();
+  const mockOnCompareModeUnselect = jest.fn();
+
+  describe('onChange function triggers mapDispatchToProps functions to notify and update the Redux store', () => {
```

Comment:
We should be describing a method e.g. `describe('onChange()'), ...`, not the behaviour - that'd be put in `it('triggers mapDispatch...`